pkgs/development/interpreters/perl/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurlBoot, buildPackages
+{ lib, stdenv, fetchurl, buildPackages
```

Comment:
I feel this is a degradation in clarity, making it clear this is a super low dependency package. Do you think otherwise?

Reply:
Well, I see your point, but I think uniformity of package expressions should be preferable to low-level implementation details, and `fetchurlBoot` is such an implementation detail.